### DISCUSSION TITLE
Provide --argument-file to allow argument blobs > 128kB

### DISCUSSION THREAD

**Discussion by pakhomov-dfinity** [dfinity/sdk#3010] [Feature Requests]
E.g. bitcoin-canister uploader needs ~800kB of arguments blob that is impossible to pass through command line arguments

**Comment by viviveevee** [comment]
Which command are you looking at? `dfx canister call` already supports an argument file

**Reply by pakhomov-dfinity** [reply]
`dfx canister install`

**Comment by ghost** [comment]
implemented `dfx canister install --argument-file <file>` in https://github.com/dfinity/sdk/pull/3431
